swift/Tests/Blockchains/FIOTests.swift

```diff
@@ -102,6 +102,43 @@ class FIOTests: XCTestCase {
         let expectedJson: String =
 """
 {"compression":"none","packed_context_free_data":"","packed_trx":"15c2285e2d2d23622eff0000000001003056372503a85b0000c6eaa664523201102b2f46fca756b200000000a8ed3232bd010f6164616d4066696f746573746e657403034254432a626331717679343037347267676b647232707a773576706e6e3632656730736d7a6c7877703730643776034554482a30786365356342366339324461333762624261393142643430443443394434443732344133613846353103424e422a626e6231747333646735346170776c76723968757076326e306a366534367135347a6e6e75736a6b39730000000000000000102b2f46fca756b20e726577617264734077616c6c657400","signatures":["SIG_K1_K85BxXzJwvjPs3mFeKatWSjBHuMXTw634RRtf6ZMytpzLCdpHcJ7CQWPeXJvwm7aoz7XJJKapmoT4jzCLoVBv2cxP149Bx"]}
+"""
+
+        XCTAssertEqual(out.json, expectedJson)
+    }
+
+    func testTransferTokens() {
+        let chainId = Data(hexString: "4e46572250454b796d7296eec9e8896327ea82dd40f2cd74cf1b1d8ba90bcd77")!
+        let privateKey = PrivateKey(data: Data(hexString: "ba0828d5734b65e3bcc2c51c93dfc26dd71bd666cc0273adee77d73d9a322035")!)!
+        let publicKey = privateKey.getPublicKeySecp256k1(compressed: false)
+        let address = AnyAddress(publicKey: publicKey, coin: .fio)
+
+        let chainParams = FIOChainParams.with {
+            $0.chainID = chainId
+            $0.headBlockNumber = 50000
+            $0.refBlockPrefix = 4000123456
+        }
+        let transferTokensAction = FIOAction.TransferTokens.with {
```

Comment:
can we just call `transfer` ?

Reply:
I followed the FIO API call namings, hence TransferTokens.

Reply:
I'm more curious about our guidelines, which one would make sense for consistency

Reply:
Renamed.